dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -82,12 +82,12 @@ using criteria::CriteriaCollection;
  */
 class LibXml2 {
  public:
-  LibXml2() {
+  explicit LibXml2(bool multiThreadingMode) {
+    if (!multiThreadingMode)
       xmlInitParser();
   }
 
   ~LibXml2() {
-      xmlCleanupParser();
```

Comment:
why this delete ?

Reply:
xmlCleanupParser is not required, it is used only to remove false positive memory leak from valgrind.